gen3utils/main.py

```diff
@@ -59,14 +61,12 @@ def validate_manifest(manifest_files):
 def validate_etl_mapping(etl_mapping_file, manifest_file):
```

Comment:
we need to throw an error at the end if the validation failed, so that the travis check will fail